src/applications/letters/utils/helpers.jsx

```diff
@@ -180,6 +180,64 @@ export const optionsToAlwaysDisplay = [
   BENEFIT_OPTIONS.serviceConnectedPercentage,
 ];
 
+// This is a new option map that assumes the only letter making use of it
+// will be the new Benefit Summary and Service Verification letter. This letter
+// is Veteran-only, so there is no dependent option for the first iteration.
+// We are leaving the dependent option in place, because there may be a future
+// enhancement to offer options in the dependent Benefit Summary letter.
+const benefitSummaryLetterLabels = {
```

Comment:
I created a new, simpler lookup table for our friendly checkbox labels. Currently we are only applying these labels to Veteran benefit summary letters, but we can easily extend it if and when we start applying these options to non-Veteran dependent letters.